app/transformers/ATSCalculations.scala

```diff
@@ -56,12 +56,12 @@ trait ATSCalculations extends DoubleUtils with Logging {
     else taxableGains - get(CgAnnualExempt)
 
   def totalCapitalGainsTax: Amount =
-    getWithDefaultAmount(LowerRateCgtRPCI) +
+    (getWithDefaultAmount(LowerRateCgtRPCI) +
       getWithDefaultAmount(HigherRateCgtRPCI) +
       get(CgDueEntrepreneursRate) +
       get(CgDueLowerRate) +
-      get(CgDueHigherRate) -
-      get(CapAdjustment)
+      get(CgDueHigherRate) +
+      get(CapAdjustment)).max(0)
```

Comment:
Bit confused with this (likely just from a lack of context).
Why is the CapAdjustment being added rather than subtracted now?
Is it because the max amount it can be is 0? Not sure why it was subtracted before in that case

Reply:
because subtracting the value is wrong and is a bug.

The max zero is just to set any negative value to zero.

Those 2 elements are verified using the tests cases provided.